src/utils/withRTLExtension.js

```diff
@@ -10,15 +11,16 @@ const styleDefRegex = /\.[^{}]+\{[^{}]+\}/g;
  * handler adds that extra style definition to the <style /> tag created by Aphrodite.
  */
 function directionSelectorHandler(selector, baseSelector, generateSubtreeStyles) {
-  if (selector !== RTL_SELECTOR) {
+  if (selector !== RTL_SELECTOR && selector !== LTR_SELECTOR) {
     return null;
   }
 
   const generated = generateSubtreeStyles(baseSelector);
+  const prefix = selector === RTL_SELECTOR ? '[dir="rtl"]' : '[dir="ltr"]';
```

Comment:
Do I understand this correctly that this change will cause all of the ltr style selectors to include this `[dir="ltr"]` as part of their selector?

Reply:
Yes.

Reply:
Hmm, that's a bit of a bummer because it makes all of our selectors slower, and it increases the size of the stylesheet, which might be an issue for AMP. cc @gilbox 

I think I understand why this is required, but it might be worth dropping a comment explaining why we always add this to the selector.